sql/_31_cherry/issue_22054_on_update/cases/trigger_002.answer_cci

```diff
@@ -0,0 +1,606 @@
+===================================================
+0
+===================================================
+0
+===================================================
+1
+===================================================
+0
+===================================================
+0
+===================================================
+1
+===================================================
+if(b is not null , 'ok', 'nok')    
+ok     
+
+===================================================
+if(c is not null , 'ok', 'nok')    
+ok     
+
+===================================================
+if(e is not null , 'ok', 'nok')    
+ok     
+
+===================================================
+if(f is not null , 'ok', 'nok')    
+ok     
+
+===================================================
+if(g is not null , 'ok', 'nok')    
+ok     
+
+===================================================
+if(h is not null , 'ok', 'nok')    
+ok     
+
+===================================================
+if(i is not null , 'ok', 'nok')    
+ok     
+
+===================================================
+0
+===================================================
+0
+===================================================
+a    b    c    d    e    f    g    h    i    total    
+1     null     null     null     null     null     null     null     null     0     
+
+===================================================
+0
+===================================================
+0
+===================================================
+0
+===================================================
+0
+===================================================
+1
+===================================================
+0
+===================================================
+0
+===================================================
+0
+===================================================
+0
+===================================================
+if( CURRENT_TIMESTAMP -d>=0 and  CURRENT_TIMESTAMP -d<10, 'ok', 'nok')    
+nok     
```

Comment:
It's better to add the issue link as a comment at https://github.com/CUBRID/cubrid-testcases/blob/develop/sql/_31_cherry/issue_22054_on_update/cases/trigger_002.sql#L50.
Since we have to add the 'nok' as the cci answer, others may confuse about it. Add the issue link will help others to understand it.